docs/site/submitting_a_pr.md

```diff
@@ -0,0 +1,495 @@
+---
+title: Submitting a pull request to LoopBack 4
+lang: en
+keywords: contributing, LoopBack community, pull request, PR, loopback
+sidebar: contrib_sidebar
+permalink: /doc/en/lb4/submitting_a_pr.html
+---
+
+## How to submit a pull request (PR) for LoopBack 4
+
+This tutorial is about how to submit a pull request (PR) for LoopBack 4 while
+following our conventions and requirements.
+
+Pick `Expert Instructions` or `Beginner Instructions` mode based your experience
+with GitHub's PR flow.
+
+To contribute high-quality code/docs, please read
+[Contributing to LoopBack](https://loopback.io/doc/en/contrib/index.html),
+[Contributing code](https://loopback.io/doc/en/contrib/code-contrib-lb4.html),
+and [Contributing to docs](https://loopback.io/doc/en/contrib/doc-contrib.html),
+before diving into the PR process.
+
+### Expert Instructions
+
+In addition to your knowledge about GitHub and creating a pull request, we have
+specific conventions and requirements you need to follow when submitting a pull
+request for LoopBack 4.
+
+#### 1. Commits to local repository
+
+Our commit messages are formatted according to
+[Conventional Commits](https://conventionalcommits.org/).
+
+Please read the
+[Commit Message Format](https://loopback.io/doc/en/contrib/code-contrib-lb4.html#commit-message-guidelines)
+guidelines to correctly format your commit messages.
+
+To change an existing commit please refer to
+[Changing a commit message](https://help.github.com/articles/changing-a-commit-message/).
```

Comment:
We should instruct contributors to run `npm install` in `loopback-next` after `git clone`. It will automatically set up git `commit-msg` hooks to check the conventions and block invalid messages.